projects/angular/src/data/_tables.clarity.scss

```diff
@@ -444,29 +246,15 @@
     th,
     td {
       // nudge down to force row to 24px height with border
-      padding-top: var(
-        --clr-table-cell-compact-padding-vertical,
-        table-variables.$clr-table--compact-verticalPadding + variables.$clr_baselineRem_1px
-      );
-      padding-bottom: var(
-        --clr-table-cell-compact-padding-vertical,
-        table-variables.$clr-table--compact-verticalPadding
-      );
+      padding: table-variables.$clr-table-cell-compact-padding-vertical 0
```

Comment:
We still want to keep left and right paddings, but entire `.table.table-compact {` section should be deleted, because the compact rules are handled by `basic-table(` mixin.